packages/openid4vc/src/shared/utils.ts

```diff
@@ -72,7 +83,9 @@ export async function getSphereonSuppliedSignatureFromJwtIssuer(
       return signature
     },
     alg: alg as unknown as SigningAlgo,
+    // @ts-expect-error: should be fixed as it is different for X.509
     did,
+    // @ts-expect-error: should be fixed as it is different for X.509
     kid,
```

Comment:
Hmm we shouldn't mere it like this. Do we want to wait until all the updates to Sphereon libs are merged/released? Or can we temp disable x5c until this is fixed so we can already merge this work? Woudln't be a bummer to keep this PR open for too long